src/encoded/static/components/static-pages/HomePage.js

```diff
@@ -43,10 +43,12 @@ export default class HomePage extends React.PureComponent {
      * @returns {Element} A React <div> element.
      */
     render() {
+        const { windowWidth, context, session } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['windowWidth' is missing in props validation (react/prop-types)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4476879)